technical-vision.md

```diff
@@ -1,89 +1,26 @@
-
-### OpenSSF is an influential advocate for efforts that are aligned with its mission
-- Support development and use of meaningful, actionable standards (e.g. SBOM formats, MFA?)
-- Catalyze improvements / action through targeted investment
-- ...
-
-> Approaches to achieve the above statement:
-> - Oversees time-bound projects to advocate for specific outcomes, e.g.
->     - Great MFA distribution project
->     - Sponsor of audits (via OSTIF & others) for critical projects
-> - TAC endorsement of specific standards-track efforts, to be incorporated across other OpenSSF foundation workstreams
->     - Direct funding / staffing on tooling and adoption efforts (e.g. SPDX-Python)?
->     - _question_: would it make sense to mandate use of the standard within foundation-hosted projects?
-> - (proposed) Engagement across platforms where developers already are (meetups, conferences, StackOverflow, etc)
->     - _idea_: should OpenSSF hire dev-rel resources to staff this?
-
-### Consumers of OSS can leverage clear & consistent trusted signals to better understand the security profile of open source content:
-- Provenance of source code & artifacts
-- Security posture of projects and artifacts
-- Community health
-- Maintainer intent (e.g. project archived, not responsive to PRs, etc)
-- Consumer demand / criticality score(?)
-- ...
-
-> Approaches to achieve the above statement:
-> - OpenSSF hosts projects that produce these signals and/or make them trustworthy and actionable
->     - OpenSSF funds the adoption of its projects (via SOS)
-> - OpenSSF hosts working groups that provide guidance on how to make use these signals
-> - Badging programs for projects
-> - End Users WG
-
-### Producers of OSS (at all skill levels; also explicitly are consumers of OSS through dependencies) have access to:
-- Best practices guides, education, etc
-- Extremely low-friction tooling to make security processes less onerous, more accurate, and trusted (burden of maintenance offloaded through opt-in automation)
+### OpenSSF is a well-respected strategic partner to large OSS foundations and individual OSS projects
+- OpenSSF is clearly positioned within broader ecosystem with a clear, altruistic charter
+- Periodically review the value proposition of OpenSSF with key stakeholders
+  - Joint OKRs on shared execution, utilization of OpenSSF-managed tools & content
+  - Roadmap alignment: if overlap exists between OpenSSF and another stakeholder, either publicly document rationale OR create/document/execute a convergence plan
+- OpenSSF speaks with a trusted voice to consumers, producers, and the broader market
+
+### OpenSSF is an influential advocate for mutually-beneficial external efforts and an educator of policy decision makers
+- Support creation & use of meaningful, actionable standards (e.g. SBOM formats, MFA)
+- OpenSSF acts as a catalyst for change and improving “secure by default”
```

Comment:
How is "improving secure by default" defined in order for advocacy to occur? do we mean to advocate for improving the experience of becoming secure by default? what about changing acceptable security minimums? both?

Reply:
this still needs addressed. Secure-by-default has a lot of different implementations depending on the context it is used. As written under this header, it simply applies that the entirety of secure-by-default discussions that happen with policy makers will include OpenSSF as a champion for it. None of which speaks the default security behavior, guarantees, or development patterns the OpenSSF's Mobilization Plan, Working Groups, SIGs, etc are working on.

Reply:
I'm not sure I understand your concern here. 

I would posit that the OpenSSF is already working on "secure by default" efforts & advocacy (e.g. SLSA that ensures tampering threats to the supply chain are reduced/eliminated; the Memory Safety SIG and entry in the mobilization plan advocates for eliminating entire classes of threats through using memory safe programming languages (e.g. Rust)).

The point of this bullet is to indicate that in N years, if the vision is realized, that  the foundation would have advocated for various actors in the ecosystem to improve their default security posture, and helped to catalyze efforts to make achieving that position easier and with less friction.

Reply:
@bobcallaway apologies for the confusion, you addressed it in your response: ([] for additional clarification)
> ensures tampering [and other security] threats to the supply chain, [open source projects, groups, and communities] are reduced/eliminated
> advocate various actors [maintainers, contributors, adopters] in the [open source] ecosystem to improve their default security posture, [development activities], and helped to catalyze efforts to make achieving that position easier and with less friction 

This provides more scoping around secure-by-default specific to the the activities the Foundation chooses to take on. I believe your explanation here, if integrated, would make that intent of that bullet more apparent to someone not already familiar with the OpenSSF.